scripts/trim-low-abund.py

```diff
@@ -130,36 +137,77 @@ def main():
 
         pass2list.append((filename, pass2filename, trimfilename))
 
+        screed_iter = screed.open(filename)
         pass2fp = open(pass2filename, 'w')
         trimfp = open(trimfilename, 'w')
 
         save_pass2 = 0
-        for n, read in enumerate(screed.open(filename)):
+        for n, is_pair, read1, read2 in broken_paired_reader(screed_iter):
             if n % 10000 == 0:
                 print '...', n, filename, save_pass2, read_reads, read_bp, \
                     wrote_reads, wrote_bp
```

Comment:
On Wed, Feb 11, 2015 at 08:30:30AM -0800, Michael R. Crusoe wrote:
> >          pass2fp = open(pass2filename, 'w')
> >          trimfp = open(trimfilename, 'w')
> >  
> >          save_pass2 = 0
> > -        for n, read in enumerate(screed.open(filename)):
> > +        for n, is_pair, read1, read2 in broken_paired_reader(screed_iter):
> >              if n % 10000 == 0:
> >                  print '...', n, filename, save_pass2, read_reads, read_bp, \
> >                      wrote_reads, wrote_bp
> 
> You can keep the 'wrote' on line 269; that's the correct place for the verb. For the variable a noun phrase is advised therefore use `written_reads` and `written_bp`

Uh, this is starting to get perilously close to bikeshedding.  I'll change
it for now, but when spelling is correct and intent is clear, I am -1 on
changing it.